sound/soc/sof/sof-audio.h

```diff
@@ -465,6 +465,54 @@ struct snd_sof_route {
 	void *private;
 };
 
+/**
+ * struct snd_sof_dai_dma_ops - DAI DMA ops optional by default unless specified otherwise
+ * @stream_get_dma: Function pointer to get DAI DMA data
+ * @stream_put_dma: Function pointer to put DAI DMA data
+ * @setup: Function pointer for DAI DMA setup
+ * @free: Function pointer for DAI DMA free
+ * @pre_trigger: Function pointer for DAI DMA pre-trigger actions
+ * @trigger: Function pointer for DAI DMA trigger actions
+ * @post_trigger: Function pointer for DAI DMA post-trigger actions
+ */
+struct snd_sof_dai_dma_ops {
+	void *(*stream_get_dma)(struct snd_sof_dev *sdev, struct snd_soc_dai *cpu_dai,
+				struct snd_pcm_substream *substream);
+	void (*stream_put_dma)(struct snd_sof_dev *sdev, struct snd_soc_dai *cpu_dai,
+			       struct snd_pcm_substream *substream);
```

Comment:
The 'stream' part of the callback does not add information on top of the `get`/`put_dma`, but the get/put is a bit hard to put in place. `get_dma_data()` perhaps?

Reply:
thats was I had get_dma_data/put_dma_data. This was @plbossart 's suggestion. If we have majority/consenses, I can change it whatever way you agree upon

Reply:
@ujfalusi I wanted to have a naming difference with ASoC core functions, e.g. snd_soc_dai_get_dma_data()

This is not a wrapper for those functions, so we have improve both the naming and the description. What does 'get' mean here?